### PR TITLE
Dropped PHP 7.2 support & added Travis config for PHP 8.0

### PR DESCRIPTION
**Pull request by brusch**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,13 +11,14 @@ env:
...
     include:
-        - php: '7.2'
-          env: PREFER_LOWEST='--prefer-lowest'
         - php: '7.3'
+          env: PREFER_LOWEST='--prefer-lowest'
         - php: '7.4'
           env: SYMFONY_VERSION='~4.4.0'
         - php: '7.4'
           env: SYMFONY_VERSION='~5.0.0'
+        - php: 'nightly'
```
Comment (brusch) [inline .travis.yml:20]: This needs to be changed to `8.0` as soon as it is provided by Travis.

File: .travis.yml [hunk 2]
```diff
@@ -31,7 +32,7 @@ before_install:
         fi
 
 install:
-    - composer update $COMPOSER_FLAGS $PREFER_LOWEST
+    - composer update --ignore-platform-req=php $COMPOSER_FLAGS $PREFER_LOWEST
```
Comment (brusch) [inline .travis.yml:35]: This can be reverted as soon as `friendsofphp/php-cs-fixer` is happy to install on PHP 8.0

File: .travis.yml (hunk 1 above)
Comment (brusch) [reply .travis.yml]: Done

**Comment by brusch** [comment]
@webmozart should be ready now (PHP `8.0` is available on travis) 😊 Would be nice if you finde the time for review, merge & release :) Thanks in advance!

**Comment by theofidry** [comment]
Hi @brusch, sorry couldn't check your PR sooner. I think we will have to move to GitHub Actions first since Travis kinda dropped OSS support.

If that helps though I don't mind to allow PHP8 in the composer.json although it would be at your own risks since we cannot test this at the moment

**Comment by brusch** [comment]
@theofidry I think it's fine, I haven't experienced any problems so far. So yes, would be nice to have that merged. 
Should I revert the changes in `travis.yml` or is it fine? Thanks again!

File: .travis.yml (hunk 2 above)
Comment (theofidry) [inline .travis.yml:35]: is that still necessary?

**Comment by theofidry** [comment]
>Should I revert the changes in travis.yml or is it fine?

I think we can keep it

File: .travis.yml (hunk 2 above)
Comment (brusch) [reply .travis.yml]: No, that should be fine now 😊 Fixed

**Comment by theofidry** [comment]
Thanks!

**Comment by brusch** [comment]
Thanks a lot! 👍

**Pull request merged**